CRM/Contribute/Form/Contribution/ThankYou.php

```diff
@@ -130,6 +119,20 @@ public function buildQuickForm() {
       $this->assign('taxTerm', $taxTerm);
       $this->assign('totalTaxAmount', $params['tax_amount']);
     }
+
+    if ($this->_priceSetId && !CRM_Core_DAO::getFieldValue('CRM_Price_DAO_PriceSet', $this->_priceSetId, 'is_quick_config')) {
+      $this->assign('lineItem', $this->_lineItem);
```

Comment:
@mattwire, per question from @eileenmcnaughton, why not do your type conversion / number formatting here, at the `assign`, rather than converting the `_lineItem` property of the form object itself?